test/modules/post/test/services.rb

```diff
@@ -227,7 +230,7 @@ def test_restart_start
       ret = true
 
       results = service_start(service_name)
-      ret &&= (results == Windows::Error::SUCCESS)
+      ret &&= (results == Windows::Error::SUCCESS || results == Windows::Error::SERVICE_ALREADY_RUNNING)
```

Comment:
The targeted service to restart may already be running; Previously that wasn't handled